_pages/en/capabilities.md

```diff
@@ -471,31 +464,25 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.2.2. Indicators  
 
-- How often your team monitors the stability and security of your IT system?  
-   a. Very Frequently  
-   b. Frequently  
-   c. Occasionally  
-   d. Rarely  
-   e. Very Rarely  
-   f. Never  
+Ideal: Very Frequently  
 
-- How do you notify your team about failure alerts?  
-   a. Via email  
-   b. Via PR in Github  
-   c. Via Slack  
-   d. Via Trello board  
-   e. Via phone call  
-   f. None of the above/not applicable  
+- How often your team monitors the stability and security of your IT system?  
+  a. Very Frequently (Ideal)  
+  b. Frequently  
+  c. Occasionally  
+  d. Rarely  
+  e. Very Rarely  
+  f. Never  
 
 - How often do you  face the situation when your team finds out from customers that your application or service is down?  
```

Comment:
- How often do you  face the situation when your team finds out from customers that your application or service is down?  
==> 
When you are notified that your application or service is down, how often does this information come from a customer?